src/irr/asset/CImageLoaderTGA.cpp

```diff
@@ -129,8 +126,8 @@ static void convertColorFlip(uint32_t regionBufferRowLenght, VkExtent3D imageExt
 		imageExtent.depth
 	};
 
-	auto stride = regionBufferRowLenght;
 	auto channels = getFormatChannelCount(destFormat);
+	auto stride = regionBufferRowLenght * channels;
```

Comment:
you're using `channels` wrong when in fact you should be using pixel block bytesize (TGA formats always have a 1x1 block physical size)

Reply:
my mistake, to fix

Reply:
still fucking wrong.